src/views/Home.jsx

```diff
@@ -1,16 +1,24 @@
 import './Home.css';
+import { SingleList } from '../components';
 
 export function Home({ data, setListPath }) {
+	const dataPresent = data.length !== 0;
 	return (
 		<div className="Home">
 			<p>
 				Hello from the home (<code>/</code>) page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `lists` array
-				 * so we can see which lists the user has access to.
-				 */}
+				{dataPresent
```

Comment:
It's awesome to actually see a list once I added the sharedList field to myself!

Was there a reason y'all chose to create a specific variable for the dataPresent over possibly doing it inline I guess with the mapping of the data? 

This could totally be a nitpick so as Alex put in slack I'm gonna put `NIT` haha because this is totally valid just curious.

```suggestion
{data.length !== 0 ? data.map((list, index) => {<SingleList key={index} name={list.name} path={list.path} setListPath={setListPath} /> : null}
```